aws/vpc/security_groups/ssh/main.tf

```diff
@@ -0,0 +1,31 @@
+resource "aws_security_group" "mod" {
```

Comment:
Should this be located here or under EC2?

Reply:
That makes sense. Moving it!